config.yml.tpl

```diff
@@ -20,18 +21,18 @@ providers:
     # client_id:
     # client_secret:
 
-# only for authorization with a GitHub token
+# list of repositories to watch when using authorization with a GitHub token
 repositories:
   - url: github.com/_USER_/_REPO_TO_WATCH_
     client:
       # user:
       # token:
       # minInterval: 1m
 
-# only for authorization with GitHub App
+# web interface configuration, only available if authorizing with GitHub App
 web:
   # Secret key to sign JSON Web Tokens
-  signing_key:
+  signing_key: # a non empty string
```

Comment:
why? it avoid people leaving the line empty...

Reply:
But that's already taken care of by the comment above it, and https://github.com/src-d/lookout/blob/master/cmd/lookoutd/web.go#L59. Otherwise, why don't we add `non empty string` to each and every one of the fields?

Reply:
fair enough